examples/postgresql-replication/postgresql.pp

```diff
@@ -0,0 +1,108 @@
+$password = 'sensu'
+$repl_password = 'secret'
+$pgpassword = 'password'
+$master_ip = '192.168.52.11'
```

Comment:
Should match docs and use primary instead of master

Reply:
Replaced instances of master with primary